Album/ViewControllers/AuthenticationControllers/LoginViewController.m

```diff
@@ -19,64 +19,64 @@ @interface LoginViewController ()
 @implementation LoginViewController
 
 - (void)viewDidLoad {
-	[super viewDidLoad];
-	// Do any additional setup after loading the view.
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
```

Comment:
I believe this comment line can be removed ;)

Reply:
Done!